backend/vaa-strapi/docker-compose.dev.yml

```diff
@@ -25,6 +25,11 @@ services:
       GENERATE_MOCK_DATA_ON_INITIALISE: ${GENERATE_MOCK_DATA_ON_INITIALISE}
       GENERATE_MOCK_DATA_ON_RESTART: ${GENERATE_MOCK_DATA_ON_RESTART}
       LOAD_DATA_ON_INITIALISE_FOLDER: ${LOAD_DATA_ON_INITIALISE_FOLDER}
+      AWS_ACCESS_KEY_ID: ${AWS_ACCESS_KEY_ID}
```

Comment:
These will actually not have any effect now, because the `config/development/plugins.ts` forces use of `maildev`. Maybe we could add another `env` setting to control that, e.g. `USE_MAILDEV_IN_DEVELOPMENT`?

Reply:
Okay! And I let's pass all vars for sure.